### PR TITLE
Add matplotlib.

### PR DESCRIPTION
**Pull request by LaurentMazare**
Add a matplotlib package.
This packages make it easy to interact with matplotlib using python via pyml. Wrappers are provided for the object oriented api of matplotlib.

### CODE REVIEW

**Comment by mseri** [comment]
I am looking forward to try it from owl!

File: packages/matplotlib/matplotlib.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+name:         "matplotlib"
```
Comment (mseri) [inline packages/matplotlib/matplotlib.0.1/opam]:
```suggestion
```

File: packages/matplotlib/matplotlib.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+bug-reports:  "https://github.com/LaurentMazare/ocaml-matplotlib/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-matplotlib"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-matplotlib.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-j" jobs "@install"]]
```
Comment (mseri) [inline packages/matplotlib/matplotlib.0.1/opam]:
```suggestion
build: [["dune" "build" "-j" jobs "-p" name]]
```

**Comment by mseri** [comment]
The failure is due to the reported pyml issue. Merging. Thanks

**Comment by LaurentMazare** [comment]
> I am looking forward to try it from owl!

@mseri for what it's worth I have the following in my init script for jupyter. Using `plot` in the notebook then flushes the current picture and displays it (no need for a temporary file, everything is in memory).
```ocaml
open Matplotlib;;
let plot () =
  let data = Mpl.plot_data `png in
  ignore (Jupyter_notebook.display ~base64:true "image/png" data);;

let () =
    Mpl.set_backend Agg;
    Mpl.style_use "ggplot"
;;
```

**Comment by mseri** [comment]
Thanks!

**Pull request merged**